<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>470</width>
    <height>290</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>Game over</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(32, 32, 32);</string>
  </property>
  <widget class="QLabel" name="textLeaderboard">
   <property name="geometry">
    <rect>
     <x>95</x>
     <y>20</y>
     <width>280</width>
     <height>40</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="minimumSize">
    <size>
     <width>280</width>
     <height>40</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>280</width>
     <height>40</height>
    </size>
   </property>
   <property name="font">
    <font>
     <family>TF2 Secondary</family>
     <pointsize>22</pointsize>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(40,40,40);
border-radius: 8%;
color: white;</string>
   </property>
   <property name="text">
    <string>GAME OVER</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QPushButton" name="CloseButton">
   <property name="geometry">
    <rect>
     <x>115</x>
     <y>219</y>
     <width>240</width>
     <height>50</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="minimumSize">
    <size>
     <width>200</width>
     <height>40</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>240</width>
     <height>220</height>
    </size>
   </property>
   <property name="font">
    <font>
     <family>TF2</family>
     <pointsize>25</pointsize>
    </font>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton {
background-color: rgba(255, 255, 255, 0);
border: 0;
color: rgb(150, 150, 150);
}
QPushButton:hover{
background-color: rgba(255, 255, 255, 45);
color: rgb(255,255,255);
}
</string>
   </property>
   <property name="text">
    <string>Close</string>
   </property>
  </widget>
  <widget class="QLabel" name="LevelText">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>150</y>
     <width>190</width>
     <height>50</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="minimumSize">
    <size>
     <width>0</width>
     <height>0</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>240</width>
     <height>50</height>
    </size>
   </property>
   <property name="font">
    <font>
     <family>TF2 Secondary</family>
     <pointsize>18</pointsize>
    </font>
   </property>
   <property name="cursor">
    <cursorShape>IBeamCursor</cursorShape>
   </property>
   <property name="styleSheet">
    <string notr="true">Border: 1px solid grey;
padding: 10px;
color: rgb(245, 245, 245);</string>
   </property>
   <property name="text">
    <string>Level : 1</string>
   </property>
  </widget>
  <widget class="QLabel" name="ScoreText">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>80</y>
     <width>190</width>
     <height>50</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="minimumSize">
    <size>
     <width>0</width>
     <height>0</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>240</width>
     <height>50</height>
    </size>
   </property>
   <property name="font">
    <font>
     <family>TF2 Secondary</family>
     <pointsize>18</pointsize>
    </font>
   </property>
   <property name="cursor">
    <cursorShape>IBeamCursor</cursorShape>
   </property>
   <property name="styleSheet">
    <string notr="true">Border: 1px solid grey;
padding: 10px;
color: rgb(245, 245, 245);</string>
   </property>
   <property name="text">
    <string>Score : 0</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
